### PR TITLE
Use CCS in 'create-cluster' test

### PR DESCRIPTION
**Pull request by kurtismullins**
Use CCS during `create-clusters` test to allow Users to supply their own AWS account.

### CODE REVIEW

File: cmd/ocm-load-test.go [hunk 1]
```diff
@@ -125,6 +125,30 @@ func run(cmd *cobra.Command, args []string) error {
...
+			log.Fatal("ccsAccountID is required to run the create-cluster test.")
+		}
+		if len(viper.GetString("ccsAccessKey")) == 0 {
+			log.Fatal("ccsAccessKey is required to run the create-cluster test.")
+		}
+		if len(viper.GetString("ccsSecretKey")) == 0 {
+			log.Fatal("ccsSecretKey is required to run the create-cluster test.")
+		}
+	}
+	*/
```
Comment (nimrodshn) [inline cmd/ocm-load-test.go:150]: @kurtismullins why is this commented out?

File: pkg/tests/handlers/clusters.go [hunk 2]
```diff
@@ -41,7 +47,9 @@ func generateCreateClusterTargeter(ID, method, url string) vegeta.Targeter {
 			Name(fmt.Sprintf("perf-%s-%d", id, idx)).
 			Properties(fakeClusterProps).
 			MultiAZ(true).
-			Region(v1.NewCloudRegion().ID(helpers.DefaultAWSRegion)).
+			Region(v1.NewCloudRegion().ID("us-east-1")).
+			CCS(v1.NewCCS().Enabled(true)).
+			AWS(v1.NewAWS().AccessKeyID(ccsAccessKey).SecretAccessKey(ccsSecretKey).AccountID(ccsAccountID)).
```
Comment (nimrodshn) [inline pkg/tests/handlers/clusters.go:52]: :100: !

**Comment by chentex** [comment]
This is looking awesome!

File: cmd/ocm-load-test.go (hunk 1 above)
Comment (kurtismullins) [reply cmd/ocm-load-test.go]: That was temporary while the PR was a draft.

**Comment by kurtismullins** [comment]
Closing this Pull Request as #38 provides a better implementation.

**Pull request closed without merging**